cmd/createorupdate/createorupdate.go

```diff
@@ -126,11 +131,78 @@ func createResourceGroup(conf *fakerp.Config) (bool, error) {
 	return true, err
 }
 
-func execute(ctx context.Context, log *logrus.Entry, rpc v20180930preview.OpenShiftManagedClustersClient, conf *fakerp.Config) error {
+func deployVnet(ctx context.Context, log *logrus.Entry, conf *fakerp.Config, rpc v20180930preview.OpenShiftManagedClustersClient) (string, error) {
+	vnet, err := ioutil.ReadFile("../../test/templates/" + *createPeerVnet + ".json")
```

Comment:
The preceding `../..` are for my own debug purposes.  Will need to reset this pathing before merge.